### PR TITLE
Adding ability to view logs and open log directory

### PR DESCRIPTION
**Pull request by amitnndn**
#1556 - Adding ability to open logs and view log directory on the help page.
#741 -  Fixing content address extending outside of the visible area in the card. (I am including this fix as a part of this PR since it is a small relatively small fix - 3 lines)

### Changes
* Made changes to the code to include a new section to giving users the option to open the log file or the directory that contains the log file. 
* Added some type declarations since eslint and flow checkers were failing on the file that I edited. 
* Added a word wrap while to card's ```h1``` element so that long URLs do not extend outside the visible area.

### Preview 
#1556 
screen shot 2018-06-09 at 6 30 04 pm

#741 
screen shot 2018-06-10 at 5 15 46 pm

P.S: I have tested this only on macOS Sierra and theoretically should work on other operating systems as well. Let me know if it doesn't!

### CODE REVIEW

**Comment by tzarebczan** [comment]
@amitnndn thanks for the contribution! We'll get this reviewed shortly. 

Similar to last time, please see https://lbry.io/faq/tips if you'd like a tip for your contribution :)

**Comment by amitnndn** [comment]
Thanks, @tzarebczan! I fixed another very simple bug in the app and included that as a part of this PR. Did not want to go through the trouble of creating another PR for such a simple fix. Let me know if that is alright!

File: .flowconfig [hunk 1]
```diff
@@ -20,5 +20,6 @@ module.name_mapper='^lbry\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/lbry\1'
 module.name_mapper='^rewards\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/rewards\1'
 module.name_mapper='^modal\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/modal\1'
 module.name_mapper='^app\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/app\1'
+module.name_mapper='^native\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/native\1'
```
Comment (seanyesmunt) [inline .flowconfig]: Is there a reason this was added?

Comment (amitnndn) [reply .flowconfig]: @seanyesmunt This was to resolve a flow error that was on [in line](https://github.com/amitnndn/lbry-app/blob/aa493f997bd99dd9ef620484bf3585d6604b4d0f/src/renderer/page/help/view.jsx#L6)

**Pull request merged**